Library.Console/PublishingHouse.cs

```diff
@@ -8,8 +10,32 @@ public class PublishingHouse
 
     public string Country { get; private set; }
 
-    public List<Book> Books { get; private set; } = new();
+    public HashSet<Book> Books { get; private set; } = new(); // fkc list, we dont wanna duplicates
+
+
+    public bool CheckBook(Book book) => Books.Where(e => e.ISBN == book.ISBN).Any();
+    public void AddBook(Book book)
+    {
+        if (book.PublishingHouse.Equals(this))
```

Comment:
To jest totalnie bez sensu jak większość tych equalsów, po co to wgl sprawdzać skoro można dodać książkę

Reply:
Jak wyżej